src/types.ts

```diff
@@ -884,7 +884,15 @@ export interface AppContext<T = unknown> {
   fsxaApi?: FSXAApi
 }
 
-export type RemoteProjectConfiguration = Record<string, { id: string; locale: string }>
+export type RemoteProjectConfiguration = {
+  [name: string]: {
```

Comment:
I know this syntax is the same as using a Record, but isn't using a Record the better/newer way?

Reply:
I'd say it depends. 
Record is good, if you habe arbitrary key value pairs, that do not share a common meaning.
Here, i wanted to convey the meaning behind the data